src/components/join-flow/username-step.jsx

```diff
@@ -279,7 +285,8 @@ class UsernameStep extends React.Component {
 
 UsernameStep.propTypes = {
     intl: intlShape,
-    onNextStep: PropTypes.func
+    onNextStep: PropTypes.func,
+    sendAnalytics: PropTypes.func
```

Comment:
General question that applies to all of these individual steps: we probably want to use `isRequired` on the sendAnalytics prop, right?

Reply:
(also applies to most other individual steps)